### ISSUE TITLE
Error Could not load type 'GraphQL.Http.IDocumentWriter' 

### ISSUE THREAD

**Issue by apis3445** [graphql-dotnet/graphql-dotnet#1850]
I update to the version 3.0 and I get this error
Could not load type 'GraphQL.Http.IDocumentWriter' from assembly 'GraphQL, Version=3.0.0.0, Culture=neutral, PublicKeyToken=nul
this is my startup code
`services.AddScoped<CaducidadSchema>();
            services.AddScoped<CaducidadInputType>();
            services.AddScoped<CaducidadMutation>();
            services.AddGraphQL(options =>
            {
                options.EnableMetrics = true;
                options.ExposeExceptions = true;
            });`

**Comment by joemcbride** [comment]
Hi, this is a breaking change in 3.0.

`IDocumentWriter` and several other classes/interfaces moved namespaces from `GraphQL.X` to  just `GraphQL`.

`GraphQL.Http.IDocumentWriter` -> `GraphQL.IDocumentWriter`

You will also need to reference a new nuget package, either `GraphQL.SystemTextJson` or `GraphQL.NewtonsoftJson`.

https://graphql-dotnet.github.io/docs/guides/migration3#json-parsing-and-serialization

**Comment by apis3445** [comment]
I added the 
`using GraphQL.SystemTextJson;`
but I can't find any sample on .net core with the new version and I don't know what I need to do. The code is broken in the 
`services.AddGraphQL(options =>
            {
                options.EnableMetrics = true;
                options.ExposeExceptions = true;
            });`

**Comment by Shane32** [comment]
See https://github.com/graphql-dotnet/server/issues/383

See #1847

**Comment by Zerotask** [comment]
I'm getting the same error with the following dependencies
```xml
<PackageReference Include="GraphQL" Version="3.0.0" />
<PackageReference Include="GraphQL.Server.Transports.AspNetCore" Version="3.4.0" />
<PackageReference Include="GraphQL.SystemTextJson" Version="3.0.0" />
```

For me it was a new project, not migration from 2.x to 3.x.

**Comment by Jonatthu** [comment]
You guys will need to wait until Server 3.5.0 probably by the end of the next week 
Based on @sungam3r

**Comment by sungam3r** [comment]
I will publish a new version today or tomorrow. It will be 4.0.0, not 3.5.0 since in fact we already have many breaking changes since 3.4.0 server.

**Comment by sungam3r** [comment]
Closed in favor of #1847
